ADPandABlocksApp/src/ADPandABlocks.cpp

```diff
@@ -608,16 +628,46 @@ void ADPandABlocks::pollCommandPort(){
 	{
 		epicsTimeGetCurrent(&pollStartTime);
 		this->lock();
-		processChanges("*CHANGES.ATTR?", false);
-		processChanges("*CHANGES.POSN?", true);
+        processChanges("*CHANGES?");
 		this->unlock();
 		callParamCallbacks();
 		epicsTimeGetCurrent(&pollEndTime);
 		epicsThreadSleep(0.1-epicsTimeDiffInSeconds(&pollEndTime, &pollStartTime));
 	}
 }
 
-void ADPandABlocks::processChanges(std::string cmd, bool posn)
+bool ADPandABlocks::checkIfCustomPosBusParam(std::string posBusName, std::string fieldName, std::string val) {
+    std::map<std::string, std::map<std::string, int*> >::iterator it = posBusLookup.find(posBusName);
+    if (it != posBusLookup.end()) {
+        std::map<std::string, int*>::iterator it2 = it->second.find(fieldName);
+        if (it2 != it->second.end()) {
+            for (int ind = 0; ind < NCUSTOM; ind++) {
```

Comment:
I'm worried this will affect performance as this code is run at 10Hz for every param change.
Also, this are not necessarily PosBus params, can't the custom code deal with any types of params?
Instead of doing this inner loop, can we add a customParam map that contains precisely what we need to extract in the loop?